packages/graphql-server/src/directives/makeDirectives.ts

```diff
@@ -50,9 +50,9 @@ export const makeDirectivesForPlugin = (
 export const getDirectiveName = (schema: DocumentNode) => {
   const definition = schema.definitions.find(
     (definition) => definition.kind === 'DirectiveDefinition',
-  ) as ExecutableDefinitionNode
+  ) as ExecutableDefinitionNode | undefined
 
-  return definition.name?.value
+  return definition?.name?.value
 }
```

Comment:
You okay to sanity check that this is the appropriate tweak @Tobbe?

Reply:
I'll have a look in a bit

Reply:
Yes, this looks good